python/rikai/types/geometry.py

```diff
@@ -170,6 +181,59 @@ def __eq__(self, o: Box2d) -> bool:
             self.to_numpy(), o.to_numpy()
         )
 
+    @staticmethod
+    def _verified_scale(
+        scale: Union[int, float, Tuple]
+    ) -> Tuple[Number, Number]:
```

Comment:
nit: probably `float` instead of `Number`? Since `Number` can be fractional or complex.

Reply:
Changed to `numbers.Real` , it needs to be able to handle `numpy` types